tests/test_regression.py

```diff
@@ -131,3 +132,28 @@ def test_p4_api_failed_opened(perforce_environment, mock_opened):
     log = perforce_environment.artifact_dir.join(f'{step_name}_log.txt').read()
     assert "Module sh got exit code 1" in log
     assert "Getting file diff failed due to Perforce server internal error" in log
+
+
+def test_which_universum_is_tested(docker_main, pytestconfig):
+    config = """
+from universum.configuration_support import Step, Configuration
+
+configs = Configuration([Step(name="Check python", command=["ls", "-la"])])
+"""
+    # THIS TEST PATCHES ACTUAL SOURCES!! BE CAREFUL
```

Comment:
You should either put three exclamation marks or leave just one.
In my opinion, it would be better to tone down this comment.